.github/workflows/ci-pr.yml

```diff
@@ -23,14 +16,7 @@ jobs:
       shell: bash
       timeout-minutes: 75
       run: |
-        ./mvnw -B -s .github/settings.xml clean install
-    - name: Capture Test Results
-      if: failure()
-      uses: actions/upload-artifact@v3
-      with:
-        name: Unit Tests
-        path: '**/surefire-reports/*.xml'
-        retention-days: 3
+        ./mvnw -B -s .github/settings.xml -Pasciidoctordocs,restdocs clean install
```

Comment:
This seems very specific to `asciidoctordocs,restdocs` -- will all the other tests execute? I did not know we even had these profiles. Where are they defined?

Reply:
They have been in place to ensure the rest docs execute even when skipping tests but not during a normal build.

Reply:
> will all the other tests execute?

Reply:
I will simplify this to a single profile.

Reply:
> > will all the other tests execute?

If you build with `./mvnw clean install -DskipTests -Pdocs` the docs will be generated.
If you build with `./mvnw clean install -DskipTests` the docs will be not generated and not tests executed.
If you build with `./mvnw clean install` the docs will be not generated and tests executed.